.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
-extend-ignore = E203, E402, E501, E731, E741, B950, W503, B028
+extend-ignore = E203, E402, E501, E731, E741, B950, W503, B028, B907
```

Comment:
["Rename B028 to B907, making it optional/opinionated."](https://github.com/PyCQA/flake8-bugbear#23117)